ydb/core/protos/local.proto

```diff
@@ -32,6 +32,16 @@ message TLocalConfig {
     repeated NKikimrSchemeOp.TResourceProfile ResourceProfiles = 1;
 }
 
+message TActorSystemInfo {
```

Comment:
Currently we send a single number TotalNodeUsage, that is the maximum over pool usages (and mem usage).
What do we expect to gain from sending this detailed info? Just seems like we will simply do the calculate usage + take maximum logic on Hive side instead.